### DISCUSSION TITLE
关于测速获取的优质IP，是否可以利用openwrt里面的ddns插件附带的脚本来实现更新域名解析

### DISCUSSION THREAD

**Discussion by quanjindeng** [XIU2/CloudflareSpeedTest#335] [讨论]
大佬你好！
关于测速获取的优质IP利用ddns更新到域名解析，我想是不是能够利用openwrt里面的ddns插件附带的脚本来实现。
我提取了里面的文件，发现都是通过传入几个变量就能完成域名更新（一部分域名服务商是直接通过http地址更新解析，一部分是通过相应的脚本更新解析），应该可以整合一下，不用重复造轮子。
麻烦大佬了，如有时间可否看下呢，提取的文件地址在这[ddns.zip](https://quanjin.tk/ddns.zip)

**Comment by XIU2** [comment]
我没有 openwrt，对这个了解有限，以前有个人问过，我也是远程协助帮他搞了搞。

**Comment by quanjindeng** [comment]
发现这个项目今天已经更新支持阿里ddns了，issue就关闭了，感谢！
[luci-app-cloudflarespeedtest](https://github.com/mingxiaoyu/luci-app-cloudflarespeedtest)
